cmd/cel-shed/eds_store_stress.go

```diff
@@ -0,0 +1,99 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"math"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/celestiaorg/celestia-node/libs/edssser"
+	"github.com/celestiaorg/celestia-node/nodebuilder"
+	"github.com/celestiaorg/celestia-node/nodebuilder/node"
+)
+
+const (
+	edsStorePathFlag = "store-path"
+	edsWritesFlag    = "eds-writes"
+	edsSizeFlag      = "eds-size"
+	edsEnableLogFlag = "log"
+	edsStoreCleanup  = "cleanup"
+)
+
+func init() {
+	shareCmd.AddCommand(edsStoreStress)
+
+	edsStoreStress.Flags().String(edsStorePathFlag, "", "Directory path to use for stress test. os.TempDir by default.")
+	edsStoreStress.Flags().Int(edsWritesFlag, math.MaxInt, "Total EDS writes to make. MaxInt by default.")
+	edsStoreStress.Flags().Int(edsSizeFlag, 128, "Chooses EDS size. 128 by default.")
```

Comment:
This actually ends up being ODS size, if you look at `RandEDS`, size*size shares get generated, than the get extended: https://github.com/celestiaorg/celestia-node/blob/9c40a7f379ca4cefa738e265e7d318b2e5245ef2/share/eds/edstest/testing.go#L28

Reply:
tru, fixed